collection-scripts/gather_network_logs

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NETWORK_LOG_PATH="${BASE_COLLECTION_PATH}/network_logs/"
+
+mkdir -p ${NETWORK_LOG_PATH}/
+
+WORKERS=$(oc get nodes --no-headers -o custom-columns=":metadata.name")
+
+for WORKER in $WORKERS; do
+    SDN_POD=$(oc -n openshift-sdn get pods --no-headers -o custom-columns=":metadata.name" --field-selector spec.nodeName=$WORKER -l app=sdn)
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "iptables-save -c" > ${NETWORK_LOG_PATH}/${WORKER}_iptables
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "ovs-vsctl show" > ${NETWORK_LOG_PATH}/${WORKER}_ovs_dump
+    oc -n openshift-sdn logs $SDN_POD > ${NETWORK_LOG_PATH}/${SDN_POD}_sdn_logs
+done
```

Comment:
If a customer has 500 nodes (or more) do we really want to run this on every node? 
Can we thread the collection of these item. See what I do in [gather_service_logs script](https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_service_logs#L15-L28), with the wait command and the `PIDS` array.

Reply:
@deads2k I feel like this (the two `exec` commands) should be worked directly into must-gather the binary (over some script like this). Something like what we do with: https://github.com/openshift/must-gather/blob/master/pkg/cmd/inspect/pod.go#L102-L123

Reply:
> @deads2k I feel like this (the two `exec` commands) should be worked directly into must-gather the binary (over some script like this). Something like what we do with: https://github.com/openshift/must-gather/blob/master/pkg/cmd/inspect/pod.go#L102-L123

I don't think so.  The inspect command is about what you should do with **any** pod.  These exec calls are abut what you do for a *particular* pod.  It doesn't make sense to collect that information on all logs.

Reply:
I would say, yes we do. If we're diagnosing some kind of networking weirdness, it's likely that the client and server will be on different nodes -- and we'll want the networking dump for both of them. That's one of the more common reasons we have to send tickets back to support - because they only know half of the picture.

If there's a way to qualify which nodes should be dumped, that would be useful, but I'm not familiar enough with how the script is intended to be run to make that call.